tests/helpers/perf_run_helper.rb

```diff
@@ -787,6 +805,15 @@ def get_perf_result
       perf.log_csv
       copy_archive_files
     end
+    scp_to(master, "util/metrics/collect_metrics_files.rb", "/root/collect_metrics_files.rb")
```

Comment:
Would it be easier to run this script via Bolt?

Reply:
no, because bolt creates a tmp area and removes it after the task is done.  So if for any reason you wanted to run it again manually, the script would be gone.  And you would have to add an output dir arg to the script run (not a problem, just not easier).  And of course you add in the possibility that bolt changes or what not.